util/build_vm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script is intended to install Mininet and IPMininet into
+# a brand-new Ubuntu virtual machine,
+# to create a fully usable "tutorial" VM.
+set -e
+
+export LC_ALL=C
+
+MN_INSTALL_SCRIPT_REMOTE="https://raw.githubusercontent.com/mininet/mininet/master/util/vm/install-mininet-vm.sh"
+DEPS="python \
+      python-pip \
+      python3 \
+      python3-pip
+      git"
```

Comment:
On my `Ubuntu 16.0.4.6 server` VM running the tests fails with:

```
mininet@mininet:~/ipmininet$ sudo python setup.py test
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
zip_safe flag not set; analyzing archive contents...

Installed /home/mininet/ipmininet/.eggs/pytest_runner-4.4-py2.7.egg
running pytest
Traceback (most recent call last):
  File "setup.py", line 39, in <module>
    url='https://github.com/oliviertilmans/ipmininet'
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 970, in run_command
    cmd_obj = self.get_command_obj(command)
  File "/usr/lib/python2.7/distutils/dist.py", line 846, in get_command_obj
    cmd_obj = self.command_obj[command] = klass(self)
  File "/usr/lib/python2.7/dist-packages/setuptools/__init__.py", line 132, in __init__
    _Command.__init__(self, dist)
  File "/usr/lib/python2.7/distutils/cmd.py", line 64, in __init__
    self.initialize_options()
  File "build/bdist.linux-x86_64/egg/ptr.py", line 97, in initialize_options
  File "build/bdist.linux-x86_64/egg/ptr.py", line 108, in ensure_setuptools_version
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 943, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 834, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (setuptools 20.7.0 (/usr/lib/python2.7/dist-packages), Requirement.parse('setuptools>=27.3'))
``` 
This is why I didn't installed pip from the system in #24.

Reply:
I fixed it by adding the following code at the end of the script.

```bash
sudo pip2 install --upgrade pip
sudo apt-get remove -y python-pip
sudo pip2 install --upgrade setuptools
```